javascript/statements.json

```diff
@@ -1268,7 +1267,7 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": "15"
```

Comment:
Do you have evidence supporting this?

Reply:
I created a test at https://nicolo-ribaudo.github.io/import-assertions-supported/ (it will alert an error if assertions are not supported), but I don't have Safari 15 to run it. However, in Safari 17 it still throws.

Indirect evidence is in the notes of the 2023-01 TC39 meeting, where the Apple representative said that if the proposal wasn't demoted to Stage 2 they would ship the `assert` syntax (search for "So let me be clear probably how WebKit would deal with this" at https://github.com/tc39/notes/blob/main/meetings/2023-01/feb-02.md)